rrfs-test/validated_yamls/templates/obtype_config/atms_n20.yaml

```diff
@@ -0,0 +1,585 @@
+     - obs space:
+         name: atms_n20
+         distribution:
+           name: "@DISTRIBUTION@"
+         obsdatain:
+           engine:
+             type: H5File
+             obsfile: data/obs/ioda_atms_n20.nc
+         obsdataout:
+           engine:
+             type: H5File
+             obsfile: atms_n20_hofx.nc4
```

Comment:
Based on this PR:  https://github.com/NOAA-EMC/RDASApp/pull/241 from @guoqing-noaa , it would be better to rename the diag/hofx file to be something like: jdiag_atms_n20.nc4.

Reply:
> Based on this recent PR: #241 from @guoqing-noaa , it would be better to rename the diag/hofx file to be something like: jdiag_atms_n20.nc4.

Thanks for the suggestion. But I am not sure if we should change the name at this PR or at future PR to address this for all yaml file templates.

Reply:
I think @guoqing-noaa already changed the name of diag files for other existing yaml files.  So maybe it's good to change now?  But I am also good if we would like to do it later. Thanks!

Reply:
Thank you for the change! @xyzemc